### DISCUSSION TITLE
Normalization of energy and force labels

### DISCUSSION THREAD

**Discussion by Yuta-Yoshimoto** [deepmodeling/deepmd-kit#3853] [Q&A] [answered]
Dear DeePMD community,

I am new to DeePMD, so I have a very basic question.
In the DeePMD-kit, it looks like there is no functionality of the normalization of target properties such as energy and force labels.
In the context of machine learning, data normalization is often beneficial to stabilize learning processes and  avoid overfitting.
So I am not sure why DeePMD-kit can actually generate such highly accurate models.

I would really appreciate it if you would give me some comments.

Best,
Yuta

**Comment by wanghan-iapcm** [comment, marked as answer]
deepmd-kit assumes that the unit of the energy and length are eV and A, respectively. They usually give a good enought scale for training. 
Note that deepmd-kit removes the bias in the energy labels, but does not scale them.

**Reply by Yuta-Yoshimoto** [reply]
Thank you so much for your comment.
I expect that energy bias is changed using the following function:
https://docs.deepmodeling.com/projects/deepmd/en/master/_modules/deepmd/fit/ener.html#EnerFitting.change_energy_bias

Does this function correspond to changing the average atomic energy of each species by least-square fitting to the target total energies?

**Reply by wanghan-iapcm** [reply]
exactly.

**Reply by Yuta-Yoshimoto** [reply]
Thank you so much!
I was able to deepen my understanding of DeePMD-kit.
